packages/app-store/giphy/api/get.ts

```diff
@@ -0,0 +1,60 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { z, ZodError } from "zod";
+
+import prisma from "@calcom/prisma";
+
+import { GiphyManager } from "../lib";
+
+const getSchema = z.object({
+  url: z.string(),
```

Comment:
We could do something like this:

```suggestion
  url: z.string().regex(giphyUrlRegexp),
```

Reply:
Nice catch! 
Fixed that